set.mm

```diff
@@ -7652,17 +7661,14 @@ an illustration of the conservativity of definitions (definitions do not
   pm2.64 $p |- ( ( ph \/ ps ) -> ( ( ph \/ -. ps ) -> ph ) ) $=
     ( wn wo orel2 jao1i com12 ) ABCZDABDZAAHIBAEFG $.
 
-  $( Theorem *2.26 of [WhiteheadRussell] p. 104.  (Contributed by NM,
-     3-Jan-2005.)  (Proof shortened by Wolf Lammen, 23-Nov-2012.) $)
-  pm2.26 $p |- ( -. ph \/ ( ( ph -> ps ) -> ps ) ) $=
-    ( wi pm2.27 imori ) AABCBCABDE $.
-
   $( Theorem *2.42 of [WhiteheadRussell] p. 106.  (Contributed by NM,
      3-Jan-2005.) $)
   pm2.42 $p |- ( ( -. ph \/ ( ph -> ps ) ) -> ( ph -> ps ) ) $=
     ( wn wi pm2.21 id jaoi ) ACABDZHABEHFG $.
 
-  $( Theorem *5.11 of [WhiteheadRussell] p. 123.  (Contributed by NM,
+$(  Implication and disjunction. $)
```

Comment:
"implication" is always implied, so do you mean " Disjunction", or "Negation an disjunction" ?

Reply:
Some simple expressions connecting implication and disjunction.  I changed the title.